README.md

```diff
@@ -1,35 +1,57 @@
-# SonarQube TFS/VSTS Marketplace Extension
+SonarQube TFS/VSTS Marketplace Extension
```

Comment:
Should probably say `Azure DevOps` rather than `TFS/VSTS`.

Reply:
I put `Azure Pipelines` which seems more accurate.